hasura/metadata/actions.graphql

```diff
@@ -1,155 +1,109 @@
 type Mutation {
-  adminUpdateUser(
-    payload: AdminUpdateUserInput!
-  ): UserResponse
+  adminUpdateUser(payload: AdminUpdateUserInput!): UserResponse
```

Comment:
updated my hasura-cli version but formatting seems still to be off. any idea why @topocount ?

Reply:
@zashton if you rebase again, this will go back as it was. Honestly not sure exactly why this is happening, but a rebase seems to fix it.

The commit dates for these changes aren't making sense to me.